src/cli.zig

```diff
@@ -511,13 +485,14 @@ pub const Arguments = struct {
         else
             null;
 
-        if (args.option("--origin")) |origin| {
-            opts.origin = origin;
-        }
+        // if (args.option("--origin")) |origin| {
+        //     opts.origin = origin;
+        // }
+
+        // if (args.option("--port")) |port_str| {
+        //     opts.port = std.fmt.parseInt(u16, port_str, 10) catch return error.InvalidPort;
```

Comment:
do we not use these for anything?

Reply:
Not that I can tell. Seems like they're vestigial from `bun dev`. 

I'd also argue they *shouldn't* do anything...seems too application-level to be exposed as a CLI flag.

Reply:
imo if these are used for things that got ✂️'d, then we should completely remove it from the code, instead of being a dead comment.